tests/boards/nrf/i2c/i2c_slave/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1 @@
+CONFIG_NRFX_TWIS2=y
```

Comment:
isnt't this redundant? same for other conf files.

Reply:
I have tried removing those and the test no longer compiles.

I guess the reason is that we don't have standard Zephyr API for TWI slave. We need to use nrfx driver for TWIS then, and there is no Kconfig that would automatically enable nrfx instance in such case